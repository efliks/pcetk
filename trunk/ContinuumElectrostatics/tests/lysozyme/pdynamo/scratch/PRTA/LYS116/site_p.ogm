ON_GEOM_CENT 121 2.000000
(-2.061319 36.802833 22.132294) 101 1.000000
(-2.061319 36.802833 22.132294) 101 0.500000
(-2.061319 36.802833 22.132294) 101 0.250000
