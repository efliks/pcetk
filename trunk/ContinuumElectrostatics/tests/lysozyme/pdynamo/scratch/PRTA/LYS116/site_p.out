Starting /home/mikolaj/local/bin//my_3diel_solver for molecule named scratch/PRTA/LYS116//site_p
using the following physical conditions:
Exterior dielectric constant, epsext = 80
Solvent probe radius, solrad = 1.4
Ion exclusion layer thickness, sterln = 2
Temperature, T = 300
Ionic strength, ionicstr = 0.1
Hueckel factor (epsext*kappasq) = 0.843038
Debye length 1/kappa = 9.7414
ln(10) * kBolt * T = 0.00413389
kBolt = 5.98442e-06
conconv = 0.000602214
econv = 332.063
Bohr radius = 0.529177
Proton Charge = 1
elementary_charge = 1.60218e-19
energy_unit = 2.30707e-18
Avogadro_Number = 6.02214e+23
voltconv = 14.3996
pkconv = 241.903
  Solute interior dielectric, epsin1 = 1
  Protein interior dielectric constant, epsin2 = 4
Vacuum dielectric constant, epsvac = 1

No blab level set (so no blabbing)
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 2
WARNING: SAVanal_calc: vertex found with count = 1
WARNING: SAVanal_calc: vertex found with count = 2
Using finite difference method with lattice levels:
                   Centering Style   grid dimension    spacing
Coarsest level: ON_GEOM_CENT resolved to (-0.664701, 20.5141, 19.0512)) 121 2
Finer level:    (-2.06132, 36.8028, 22.1323) 101 1
Finer level:    (-2.06132, 36.8028, 22.1323) 101 0.5
Finest level:   (-2.06132, 36.8028, 22.1323) 101 0.25


Self energy of scratch/PRTA/LYS116//site_p in three dielectric environment = 105.174
(probably in kcal/mole, INCLUDING GRID ARTEFACTS)


Interaction energy of scratch/PRTA/LYS116//site_p with background charge set in three dielectric environment = -2.16904
(probably in kcal/mole)



Interaction energies of scratch/PRTA/LYS116//site_p with charges of given instances of sites (probably in kcal/mole).
Format: site instance energy

0   0 : 0.00289029
0   1 : 1.69997e-05
1   0 : 0.000313039
1   1 : -0.00389035
2   0 : 0.0166271
2   1 : 0.000150577
3   0 : 0.0103324
3   1 : 0.00185462
3   2 : -0.000683292
3   3 : -0.00808868
4   0 : -0.000690995
4   1 : -0.0433201
5   0 : 0.00333508
5   1 : -0.0381727
6   0 : -0.0617637
6   1 : -0.545431
7   0 : 0.0171738
7   1 : 0.000388074
8   0 : -0.0029429
8   1 : -0.111415
9   0 : -0.00225657
9   1 : -0.042604
10   0 : -0.00363674
10   1 : -0.0686753
11   0 : 0.00123007
11   1 : -0.0105237
12   0 : -0.000189417
12   1 : -0.0105141
13   0 : -5.23716e-05
13   1 : -0.0037824
14   0 : 0.0183187
14   1 : 0.000868444
15   0 : 0.0154825
15   1 : 0.000179436
16   0 : 0.00428633
16   1 : -0.0596999
17   0 : 210.349
17   1 : 70.8148
18   0 : 0.00307901
18   1 : -0.0590554
Total runtime: 15.82 s user 0.12 s system 
